hedera-node/src/main/java/com/hedera/services/grpc/marshalling/AliasResolver.java

```diff
@@ -42,6 +44,11 @@ public class AliasResolver {
     private int perceivedInvalidCreations = 0;
     private Map<ByteString, EntityNum> resolutions = new HashMap<>();
 
+    /* ---- temporary token transfer resolutions map containing the token transfers to alias, is needed because a
+    token is allowed to be repeated in multiple token transfer lists, but not be repeated in a single token transfer
```

Comment:
Typo I think, you mean "...because an **alias** is allowed to be repeated...", right?

Reply:
yeah , right ! Will fix that 😐